src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -3228,20 +3228,22 @@ private IEnumerable<PSModuleInfo> CreateFakeModuleObject(IEnumerable<ModuleSpeci
                 {
                     newManifestInfo.PrivateData = manifestInfo.PrivateData;
                 }
-
-                // Assign the PowerShellGet related properties from the module manifest
-                foreach (var tag in manifestInfo.Tags)
+                else
                 {
-                    newManifestInfo.AddToTags(tag);
+                    // Assign the PowerShellGet related properties from the module manifest
+                    foreach (var tag in manifestInfo.Tags)
+                    {
+                        newManifestInfo.AddToTags(tag);
+                    }
+
+                    newManifestInfo.ReleaseNotes = manifestInfo.ReleaseNotes;
+                    newManifestInfo.ProjectUri = manifestInfo.ProjectUri;
+                    newManifestInfo.LicenseUri = manifestInfo.LicenseUri;
+                    newManifestInfo.IconUri = manifestInfo.IconUri;
```

Comment:
Hmm, I wonder why we do this at all if PrivateData is null?

Reply:
Which one, `$manifestInfo`?

Tbh. I'm confused in general by this massive method as mentioned in https://github.com/PowerShell/PowerShell/issues/17759#issuecomment-1193338559. Hopefully the PowerShell-team can elaborate 🙂